enhancements/kube-apiserver/stability-api-removal-notifications.md

```diff
@@ -54,6 +53,29 @@ Fire an `APIRemovedInNextReleaseInUse` alert when an API that will be removed in
 On EUS releases, also fire an `APIRemovedInNextEUSReleaseInUse` alert when an API that will be removed in the next
 extended support release is in use.
 
+Introduce a `DeprecatedAPIRequest` API to track users of deprecated APIs.
+
+Example:
+
+```yaml
+version: tbd/v1alpha1
+kind: DeprecatedAPIRequest
+meta-data:
+  name: 'flowschemas.flowcontrol.apiserver.k8s.io.v1alpha1'
```

Comment:
name should match the format for a CRD name

Reply:
CRD name is typically only `resource.group` and can contain many versions. What if we have multiple deprecated version of an API at once?